docs/rules/consistent-test-it.md

```diff
@@ -1,14 +1,26 @@
 # Have control over `test` and `it` usages (`consistent-test-it`)
 
+<!-- RULE_NOTICE -- Generated by `yarn tools:regenerate-docs` -->
+
+<!-- prettier-ignore -->
```

Comment:
It should be safe to apply prettier to these and I think it's better to be consistent as much as possible

Reply:
I was able to remove the prettier ignore comments.